external_libraries/CMakeLists.txt

```diff
@@ -6,6 +6,11 @@ if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/nova-simd/vec.hpp)
 			" https://github.com/supercollider/supercollider/wiki/git-cheat-sheet#working-with-submodules")
 endif()
 
+if(BUILD_PORTAUDIO)
+  message(STATUS "PortAudio: building static library")
+  add_subdirectory("${CMAKE_SOURCE_DIR}/external_libraries/portaudio")
```

Comment:
would also recommend 4 spaces here since the block above is also 4 and i do plan on reformatting this eventually, but up to you